frontend/common/src/components/form/Select/SelectFieldV2.tsx

```diff
@@ -79,6 +79,25 @@ const LocalizedNoOptionsMessage = <
     </components.NoOptionsMessage>
   );
 };
+/**
+ * One-off hook to add default messages to validation rule object in place of booleans.
+ *
+ * @param rules initial rule object
+ * @returns modified rule object
+ */
+export const useRulesWithDefaultMessages = (rules: RegisterOptions = {}) => {
+  const { formatMessage } = useIntl();
+  const rulesWithDefaults = { ...rules };
+
+  if (rules.required === true) {
+    rulesWithDefaults.required = formatMessage({
+      defaultMessage: "This field is required.",
+      description: "The default validation message when a field is required",
+    });
+  }
```

Comment:
We've already got this message somewhere we can import from

```jsx
import { errorMessages } from "../../../messages";
```

```suggestion
  if (rules.required === true)
    rulesWithDefaults.required = formatMessage(errorMessages.required);
```

Reply:
done